src/app-layout/drawer/interfaces.ts

```diff
@@ -29,15 +29,15 @@ export interface DesktopDrawerProps {
   onClick?: (event: React.MouseEvent) => void;
   onLoseFocus?: (event: React.FocusEvent) => void;
   drawers?: {
-    items: Array<DrawerItem>;
-    activeDrawerId: string | undefined;
-    onChange: (changeDetail: { activeDrawerId: string | undefined }) => void;
+    items: Array<DrawerItem | AppLayoutProps.Drawer>;
```

Comment:
I had to add this union type in lots of places to avoid a typescript error, since the types are slightly different.

Reply:
I will take over this PR and push updates today or tomorrow